thrust/thrust/system/cuda/detail/core/triple_chevron_launch.h

```diff
@@ -40,6 +40,20 @@
 
 #include <cuda/cmath>
 
+// True, when programmatic dependent launch is available, otherwise false.
+#define _THRUST_HAS_PDL _CCCL_CUDACC_AT_LEAST(11, 8)
+#if _THRUST_HAS_PDL
+// Waits for the previous kernel to complete (when it reaches its final membar). Should be put before the first global
+// memory access in a kernel.
+#  define _THRUST_PDL_GRID_DEPENDENCY_SYNC() NV_IF_TARGET(NV_PROVIDES_SM_90, cudaGridDependencySynchronize();)
+// Allows the subsequent kernel in the same stream to launch. Can be put anywhere in a kernel.
+// Heuristic(ahendriksen): put it after the last load.
+#  define _THRUST_PDL_TRIGGER_NEXT_LAUNCH() NV_IF_TARGET(NV_PROVIDES_SM_90, cudaTriggerProgrammaticLaunchCompletion();)
+#else
+#  define _THRUST_PDL_GRID_DEPENDENCY_SYNC()
+#  define _THRUST_PDL_TRIGGER_NEXT_LAUNCH()
+#endif // _THRUST_HAS_PDL
+
```

Comment:
Those macros are only used within thrust for the moment but they are not specific to thrust. 

Should we rather move them into CCCL and name them `_CCCL_PDL_MEOW`

Reply:
I moved them to `cuda/std/__cccl/cuda_capabilities.h` as discussed offline.

Reply:
Discussed offline. We move it to `__cccl/cuda_capabilities.h`